draft-ietf-quic-transport.md

```diff
@@ -650,8 +650,18 @@ described in {{packet-numbers}}.  The client increments the packet number from
 its previous packet by one for each Handshake packet that it sends (which might
 be an Initial, 0-RTT Protected, or Handshake packet).
 
-The payload of this packet contains STREAM frames and could contain PADDING and
-ACK frames.
+Servers MUST NOT send more than three Handshake packets without receiving a
+packet from a verified source address.  If the server expects to generate more
```

Comment:
You should probably explain that the server is only able to verify a client source address after receiving an address validation token or the final cryptographic handshake message from the client (in TLS, we can tell earlier, but that's not a transport-level guarantee).